lib/package-managers/npm.js

```diff
@@ -107,25 +107,67 @@ module.exports = {
             npm.commands.listAsync(args || [], true); // silent:true
     },
 
-    latest: cint.partialAt(view, 1, 'dist-tags.latest'),
+    latest: function (packageName, currentVersion, includePrerelease) {
+        return view(packageName, 'dist-tags.latest')
+            .then(function (version) {
+                if (versionUtil.getPrecision(version) === 'release' &&
```

Comment:
Almost! If getPrecision is not `'release'`, we always want to return the dist-tags version.